index.js

```diff
@@ -0,0 +1,19 @@
+const userLogin = document.querySelector("#user-login");
+
+const setName = (name) => {
+    if (name) {
+        userLogin.innerHTML = `Hello, ${name}!`;    
+    }
+}
+
+const fetchData = () => {
+    fetch('https://staging-api.realdevsquad.com/users/self')
+    .then(res => res.json())
+    .then(res => {
+        setName(res.first_name)
```

Comment:
Nit: Maybe a more descriptive name next time? `setNameIfFound`

More read:
https://google.github.io/eng-practices/review/reviewer/standard.html
https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak

Reply:
Done. Thank you!